crates/swc/tests/tsc-references/es6/computedProperties/computedPropertyNames47_ES5/input.ts/es2015.2.minified/output.js

```diff
@@ -1,4 +1,4 @@
-var E1, E2, E11, E21;
-(E1 = E11 || (E11 = {
-}))[E1.x = 0] = "x", (E2 = E21 || (E21 = {
-}))[E2.x = 0] = "x", E11.x || E21.x;
+var E1, E2, E1, E2;
```

Comment:
I don't understand why it does this, but it's ok because of `var` and won't error at runtime (it's at least better than before because it will assign to `E1` which might be used in the global scope).

Feel like this should be improved to not create the same declaration name twice though.